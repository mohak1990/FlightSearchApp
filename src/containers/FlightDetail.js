import React from 'react';
import FlightDetail from '../components/flightDetail'
import { connect } from 'react-redux'
import {  } from '../modules/filterPanel'

class FlightDetailContainer extends React.Component {

  render() {
    return (
        <FlightDetail />
    );
  }
}

const mapStateToProps = (state, ownProps) => ({

})

const mapDispatchToProps = function(dispatch, ownProps){

  return {

  }
}

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(FlightDetailContainer)
