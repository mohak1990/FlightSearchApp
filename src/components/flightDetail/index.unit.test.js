import React from 'react';
import FlightDetail from './index';
import { mount, shallow } from 'enzyme';
import ShallowRenderer from 'react-test-renderer/shallow'

test('Snapshot testing for FlightDetail component', () => {
  const renderer = new ShallowRenderer()
  const result = renderer.render(<FlightDetail data = {[]} />)
  expect(result).toMatchSnapshot()
});

it('check if DetailsForm component exist', () => {
    const FlightDetailComponent = shallow(<FlightDetail data = {[]} />);
    expect(FlightDetailComponent.find('DetailsForm')).toHaveLength(1);
});
