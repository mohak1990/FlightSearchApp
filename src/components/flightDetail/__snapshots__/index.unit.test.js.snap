// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing for FlightDetail component 1`] = `
<div>
  <DetailsForm
    isMultiple={false}
    timeTaken="NaNh NaNm"
  />
</div>
`;
